cfgov/unprocessed/js/modules/ExternalSite.js

```diff
@@ -0,0 +1,82 @@
+/* ==========================================================================
+   External Site Initialization
+   Used on at least `/external-site/`.
+   ========================================================================== */
+
+'use strict';
+
+var atomicCheckers = require( '../modules/util/atomic-checkers' );
+
+/**
+ * ExternalSite
+ * @class
+ *
+ * @classdesc Initializes a new ExternalSite instance.
+ *
+ * @param {HTMLElement} element DOM Element.
+ */
+function ExternalSite( element ) {
+
+  var BASE_CLASS = 'external-site_container';
```

Comment:
Yes it could. I see what you mean, so like why not just have `external-site_container` literally placed in `atomicCheckers.validateDomElement( element, 'external-site_container', 'ExternalSite' );`? So that's just for consistency with https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/unprocessed/js/organisms/ExpandableGroup.js#L24 so that the top-level class of the atomic unit is at the top of the constructor. 

It could be like this too:

```
  var BASE_CLASS = 'external-site';
  …

  var _dom =
    atomicCheckers.validateDomElement( element, BASE_CLASS + '_container', 'ExternalSite' );
  var _durationEl = _dom.querySelector( '.' + BASE_CLASS + '_reload-container' );
  var _directEl = _dom.querySelector( '.' + BASE_CLASS + '_proceed-btn' );
```

That way if the HTML is updated with a new class, it'll cascade with one update.
Let me know if that or something else would be preferable to what's here.